### ISSUE TITLE
Declaration parsing fails with single quotes

### ISSUE THREAD

**Issue by jcushman** [nashwaan/xml-js#8]
I'm running version 0.9.7. This works:

```
> var convert = require('xml-js');
> convert.xml2js('<?xml version="1.0" encoding="utf-8"?>')
{ declaration: { attributes: { version: '1.0', encoding: 'utf-8' } } }
```

But the same fails with single quotes for the attributes:

```
> convert.xml2js("<?xml version='1.0' encoding='utf-8'?>")
{ declaration: { attributes: { undefined: undefined } } }
```

I haven't found a formal reference, but from forum posts it sounds like both single or double quotes should be valid for declarations.

Thanks!

**Comment by nashwaan** [comment]
@jcushman Sorry about this limitation. This is actually coming from the underlying parser engine that I am using for this converter tool.
However, for _very simple_ cases, you can use following code to prepare the xml input before applying this parser.
```js
xml = xml.replace(/"/g, '\'').replace(/'/g, '"')
```

**Comment by BNUZ-YLSY** [comment]
I think that you should put this in the "`README.MD`".  The only 4 little `'` in my file cause the convert failure, it blocked my work for about one day...

**Comment by nashwaan** [comment]
@Stark-X it turned out that this was a problem with my library not the underlying parser. Today, another user faced the same issue and he [managed to identify the reason and solved it](https://github.com/nashwaan/xml-js/issues/34).

The fix is now released under v1.5.2 of this library.
I am extremely sorry for overlooking this issue.

**Comment by nashwaan** [comment]
My apologies to @jcushman as well.
